Tools/Frame_params/Holybro-S500.param

```diff
@@ -0,0 +1,32 @@
+#Note: Holybro S500 param for ArduPilot Copter 4.1
+ACRO_YAW_P,1.5
+ANGLE_MAX,3500
+ATC_ACCEL_P_MAX,70000
+ATC_ACCEL_R_MAX,70000
+ATC_ACCEL_Y_MAX,12000
+ATC_ANG_PIT_P,11
+ATC_ANG_RLL_P,11
+ATC_ANG_YAW_P,7.2
+ATC_RAT_PIT_D,0.00602
+ATC_RAT_PIT_I,0.17
+ATC_RAT_PIT_P,0.17
+ATC_RAT_RLL_D,0.00602
+ATC_RAT_RLL_I,0.17
+ATC_RAT_RLL_P,0.17
+ATC_RAT_YAW_FLTE,1
+ATC_RAT_YAW_I,0.06
+ATC_RAT_YAW_P,0.6
+BATT_AMP_PERVLT,39.877
+BATT_CURR_PIN,1
+BATT_MONITOR,4
+BATT_VOLT_MULT,18.182
+BATT_VOLT_PIN,0
+BRD_SAFETYENABLE,0
+FRAME_CLASS,1
+FRAME_TYPE,1
+INS_GYRO_FILTER,40
+MOT_BAT_VOLT_MAX,12.6

```

Comment:
the S500 web page seems to suggest 4S, but this is for 3S. Is that right?

Reply:
yes, it is.  shoot..  I think I've got a 4S battery so I'll test with that.

Reply:
might change tuning a bit too

Reply:
updated using a 4S battery.